rabbitmq/src/main/java/io/micronaut/rabbitmq/intercept/RabbitMQIntroductionAdvice.java

```diff
@@ -164,18 +164,6 @@ public Object intercept(MethodInvocationContext<Object, Object> context) {
                     }
                 });
 
-//                final Optional<Argument> messageHeadersArgument = findMessageHeadersArgument(context);
```

Comment:
Is this supposed to be commented out?

Reply:
You probably are looking at an outdated file. They are removed in the latest code.